.github/workflows/reusable_fast.yml

```diff
@@ -1,54 +1,112 @@
-# Fast builds
+# Fast build linux part is working in dockers, Windows is not
 name: FastBuild
 
 on: workflow_call
 
 permissions:
   contents: read
+  packages: read
 
 env:
   BUILD_DIR : "${{github.workspace}}/build"
   INSTL_DIR : "${{github.workspace}}/../install-dir"
 
 jobs:
-  FastBuild:
-    name: Fast builds
+  ubuntu-build:
+    name: Linux
```

Comment:
Fast Builds Linux?

Reply:
tbh I like to keep this name simple as "fastbuild" is alredy part of the workflow name...

also, we don't have any unity in the naming, so probably to be fixed globally ;)